.github/workflows/ci.yml

```diff
@@ -538,6 +539,7 @@ jobs:
             - { modules: plugin/trino-postgresql }
             - { modules: plugin/trino-raptor-legacy }
             - { modules: plugin/trino-redis }
+            - { modules: plugin/trino-redshift }
```

Comment:
this will fail.
we should have only `{ modules: plugin/trino-redshift, profile: cloud-tests }` 

let's add some comment